src/main/java/emissary/Emissary.java

```diff
@@ -105,6 +106,7 @@ protected void execute(String[] args) {
             }
             EmissaryCommand cmd = commands.get(commandName);
             dumpBanner(cmd);
+            LOG.info(GitRepositoryState.dumpVersionInfo(GitRepositoryState.getRepositoryState()));
```

Comment:
Make this line a method, and I think we're good to go. That way in an implementation we can call:
```
@Override
protected void dumpVersionInfo() {
    super.dumpVersionInfo();
    LOG.info(
       GitRepositoryState.dumpVersionInfo(
           GitRepositoryState.getRepositoryState("<implementation>.git.properties")
        )
    );
}
```